src/main/java/lihua/commands/HelpCommand.java

```diff
@@ -0,0 +1,50 @@
+package lihua.commands;
+
+import lihua.commons.Messages;
+
+/**
+ * Command class representing a command to get help.
+ */
+public class HelpCommand extends Command {
+    /** Command word for help command */
+    public static final String COMMAND_WORD = "help";
```

Comment:
Do we need COMMAND_WORD and MESSAGE_USAGE here at the same time both serving as attributes. I think you should only keep COMMAND_WORD as attribute and the other could be converted to method return.